app/frontend/courses/curricula/types/CoursesCurriculum__Grading.res

```diff
@@ -17,14 +17,13 @@ let grade = t => t.grade
 
 let pending = evaluation => evaluation |> List.exists(grading => grading.grade == None)
 
-let isFail = (passGrade, grading) =>
+let isFail = grading =>
   switch grading.grade {
-  | Some(grade) => grade < passGrade
-  | None => false
+  | Some(_) => false
+  | None => true
   }
```

Comment:
You can use the `Belt.Option` module to work with Options. We generally don't switch if we are not using the value; 
```res
let isFail = grading => Belt.Option.isNone(grading)
```

Reply:
Also I don't think the function is used, you can remove it if its not used;